middlewares/auth.js

```diff
@@ -26,5 +29,23 @@ class Strategy {
     this.GithubStrategy = passport.use(GithubStrategy);
     this.serializePassportUser = serializePassportUser(passport, user);
   }
+
```

Comment:
I don't think this method should be here, this class was meant for strategy authentications.
so verifying the token in here. should be in a different file.
but so far it looks good. nice job